Client/Frontend/Strings.swift

```diff
@@ -3981,6 +3981,21 @@ extension String {
             tableName: "Shopping",
             value: "This could take about 60 seconds.",
             comment: "Description for info card when the product is in analysis mode")
+        public static let InfoCardProductNotInStockTitle = MZLocalizedString(
+            key: "", // Shopping.InfoCard.ProductNotInStock.Title.v121
+            tableName: "Shopping",
+            value: "Product Is Not Available",
+            comment: "Title for info card when the product is not in stock or recently back in stock")
```

Comment:
I think these strings need more info - like this the user doesn't know that this is for the review checker.